platform/darwin/src/MGLEvent.mm

```diff
@@ -0,0 +1,107 @@
+#import "MGLEvent_Private.h"
+
+const MGLEventType MGLEventTypeResourceRequest = @"resource-request";
+
+#pragma mark - Value conversion
+
+// From value.md
+static id MGLJSONObjectFromMapboxBaseValue(const mapbox::base::Value &value) {
+    // Supported types are `int`, `uint`, `bool`, `double`, `array`, `object` and `string`.
+    return value.match(
```

Comment:
I'm wondering if there is another value->id converter in the repository, maybe in expression or style code?

Reply:
You're right, I think we do. I was looking at `mbgl::Value` and assuming this was different from `mapbox::base::Value`.

Reply:
@alexshalamov do you think we can reuse [this one](https://github.com/mapbox/mapbox-gl-native-ios/blob/0f6c746a447ff3da15559f8cfd6c4d045756b9d0/platform/darwin/src/MGLStyleValue.mm#L8)? Looks like we'd need to add support for `uint64_t`.

Reply:
Done.